packages/maskbook/src/plugins/Voicechat/utils/volumeMeter.ts

```
@@ -0,0 +1,58 @@
+export function VolumeMeter(context: AudioContext, opts: any, onenterframe: Function) {
+    if (typeof opts === 'function') {
+        onenterframe = opts
+        opts = null
+    }
+    opts = opts || {}
+    opts.fftSize = opts.fftSize || 32
+    opts.tweenIn = opts.tweenIn || 1.618
+    opts.tweenOut = opts.tweenOut || opts.tweenIn * 3
+
+    // @ts-ignore
```

why need ignore error?

---

plz add note to the line comment